selfdrive/ui/widgets/ssh_key.py

```diff
@@ -41,11 +39,14 @@ def __init__(self):
 
     self._refresh_state()
 
+  def get_action_width(self) -> int:
+    return self.MAX_WIDTH
+
   def _refresh_state(self):
     self._username = self._params.get("GithubUsername", "")
-    self._state = SshKeyActionState.REMOVE if self._params.get("GithubSshKeys") else SshKeyActionState.ADD
+    self._state = SshKeyState.REMOVE if self._params.get("GithubSshKeys") else SshKeyState.ADD
 
-  def _render(self, rect: rl.Rectangle) -> bool:
+  def render_action(self, rect: rl.Rectangle) -> bool:
```

Comment:
why not use generic render function?

Reply:
render_action is called in _render now, to render the right controls in the list view, it's used internal

Reply:
The render_action method is called within the base class’s _render() method. The base class _render() draws the list item’s icon, title, and description, then calls render_action() to draw the controls on the right side.